lang/en/docs/tutorials/dft/electronic/valence-band-offset.md

```diff
@@ -0,0 +1,101 @@
+# Calculate Valence Band Offset
+
+This tutorial page explains how to calculate the [valence band offset](../../../properties-directory/scalar/valence-band-offset.md) (VBO)
+based on the potential lineup method[^1][^2][^3] using [Density Functional Theory](../../../models-directory/dft/overview.md).
+For this tutorial, we consider a 2D material interface MoS<sub>2</sub>/WS<sub>2</sub> and use
+[Quantum Espresso](../../../software-directory/modeling/quantum-espresso/overview.md) as our main simulation engine.
+The content of this tutorial was also part of our 2021 webinar *2D Materials and their Electronic Properties*[^4]
+
+!!!note "Simulation engines considered in this tutorial"
+    The workflow presented in this tutorial is currently only available for
+    [Quantum ESPRESSO](../../../software-directory/modeling/quantum-espresso/overview.md).
+
+## Definitions
+
+### Valence Band Offset
+
+The [valence band offset](../../../properties-directory/scalar/valence-band-offset.md) is defined by the relative position
+of the valence band on both sides of the interface. This property is relevant to study the charge transport across
+interfaces such as semiconductor heterojunctions. Other properties related to the band profile at the interface are
+the *conduction band offset* and *Schottky barrier* (metal-semiconductor interface).
+
+### Potential Lineup Method
+This tutorial employs the potential lineup method in order to determine the valence band offset, which requires the
+calculation of the macroscopically averaged electrostatic potential and valence band maximum of the two materials.
+The valence band offset for an A/B interface can then be determined via:
+$$ \Delta E_{\mathrm{VBO}} = \Delta E_{v} + \Delta V$$
+The first term, $\Delta E_{v}$, is usually referred to as the *band structure term* and defined as the difference
+of the two valence band maxima $\varepsilon_{v}$ referenced to the macroscopically averaged electrostatic potential $\overline{V}$
+in each material:
+$$ \Delta E_{v} = (\varepsilon_{v}^{A} - \overline{V}^{A}) - (\varepsilon_{v}^{B} - \overline{V}^{B}) $$
+The second term, $\Delta V$, is determined from the lineup of the macroscopically averaged electrostatic potential in
+the interface heterostructure.
```

Comment:
Could we, perhaps, add some more explanation about the workflow? Maybe a link to the YAML source in WoDe.js, and a rough flowchart drawing?

Although there is a video, this page itself is rather dry without any visuals.

Reply:
I think the YAML source would not be much help, it is very difficult to learn from it if one doesn't know how wode.js works. I made a diagram of the workflow and embed it on the tutorial page.

Reply:
Let's at least link the two? Link to docs in wode and link to wode source in here